packages/ivass-certified-attributes-importer/src/service/tenantProcessService.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-params */
+import { tenantApi } from "pagopa-interop-api-clients";
+import { Logger } from "pagopa-interop-commons";
+import { InteropContext } from "../model/interopContextModel.js";
+
+export class TenantProcessService {
+  private readonly client: ReturnType<typeof tenantApi.createInternalApiClient>;
+
+  constructor(tenantProcessUrl: string) {
+    this.client = tenantApi.createInternalApiClient(tenantProcessUrl);
+  }
+
+  public async internalAssignCertifiedAttribute(
+    tenantOrigin: string,
+    tenantExternalId: string,
+    attributeOrigin: string,
+    attributeExternalId: string,
+    context: InteropContext,
+    logger: Logger
+  ): Promise<void> {
+    return await this.client
+      .internalAssignCertifiedAttribute(undefined, {
+        params: {
+          tOrigin: tenantOrigin,
+          tExternalId: tenantExternalId,
+          aOrigin: attributeOrigin,
+          aExternalId: attributeExternalId,
+        },
+        headers: {
+          "X-Correlation-Id": context.correlationId,
+          Authorization: `Bearer ${context.bearerToken}`,
+          "Content-Type": false,
+        },
+      })
+      .catch((err) => {
+        logger.error(
+          `Error on internalAssignCertifiedAttribute. Reason: ${err.message}`
+        );
+        throw Error(
```

Comment:
```suggestion
        throw genericError(
```
We could use the already defined `genericError` in these cases

Reply:
`genericError` is an `ApiError`, meant to be caught and mapped to an API response with code, etc. 
In these cases, where we are not coding API services but scripts or jobs, I think it's ok to use `Error` or the already-defined `genericInternalError` (or any other internal error that may suit the case)

Reply:
You're right, let's use `Error` then 👍

Reply:
Well this is a cronjob so I don't see a real benefit in using a `genericError`

Reply:
Well this is a cronjob so I don't see a real benefit in using a `genericError`